website/cloudbuild.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright 2019 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+  - name: 'gcr.io/cloud-builders/npm'
+    args: ['ci']
```

Comment:
This may need to be run in the `work` directory?

Reply:
The new scheme is targeting cloud run. I never want to see this yaml again :)

I will figure out a way to do CI shortly. (GitHub actions probably work for this, or it's trivial to make Kokoro do it also.)